sem/manager.py

```diff
@@ -221,3 +225,18 @@ def get_space(self, current_query, param_space, stdout_parsing_function,
 
     def __str__(self):
         return "--- Campaign info ---\n%s\n------------" % self.db
+
+    def check_repo_ok(self):
```

Comment:
Where is this function called? What's the usage scenario of this function?

Reply:
The function is to be called before performing additional simulations, in order to make sure that the same exact code is used for all simulations saved in the database. This prevents users from mixing up simulations that don't come from the same code.

Reply:
In this case, my opinion is to only block when the ns-3 release version is different. For the case of different HEAD commit, showing a warning message and then let users to choose whether to run further simulations.

Why?
When a user uses dev version and sync with ns-3 repo everyday, it's very hard to guarantee that the latest commit will always be the same. Also, different HEAD commit doesn't mean the simulation results are different. We shouldn't make such strong assumption.